scripting/src/main/groovy/com/powsybl/scripting/groovy/GroovyScripts.groovy

```diff
@@ -6,17 +6,31 @@
  */
 package com.powsybl.scripting.groovy
 
-import com.powsybl.afs.AfsException
-import com.powsybl.afs.AppData
-import com.powsybl.afs.FileExtension
-import com.powsybl.afs.Folder
-import com.powsybl.afs.ProjectFileExtension
-import com.powsybl.afs.ProjectFolder
+import com.powsybl.afs.*
 import org.codehaus.groovy.control.CompilerConfiguration
 
 import java.nio.charset.StandardCharsets
 import java.nio.file.Path
 
+class BuilderSpec {
+
+    private final def builder
+
+    BuilderSpec(Object builder) {
+        this.builder = builder
+    }
+
+    def methodMissing(String name, args) {
```

Comment:
Assert that name is not null

Reply:
This method is called by Groovy framework when trying to call a missing method, so there is no no need to check anything